src/components/Entry/Card/index.tsx

```diff
@@ -14,53 +13,30 @@ import cssBinder from 'styles/cssBinder';
 import styles from './styles.css';
 
 const css = cssBinder(styles);
-
-const description2IDs = (description: string) =>
-  (description.match(/"(PUB\d+)"/gi) || []).map((t) =>
-    t.replace(/(^")|("$)/g, ''),
-  );
-
 type Props = {
   data: {
     metadata: EntryMetadata;
     extra_fields?: {
       counters: MetadataCounters;
-      description: Array<StructuredDescription>;
       literature?: Record<string, Reference>;
+      short_name: string;
     };
   };
   search: string;
   entryDB: MemberDB | 'interpro';
-  showDescription?: boolean;
 };
-const EntryCard = ({
-  data,
-  search,
-  entryDB,
-  showDescription = true,
-}: Props) => {
+const EntryCard = ({ data, search, entryDB }: Props) => {
+  console.log(data.extra_fields);
   const name =
+    typeof data.extra_fields?.short_name === 'string'
+      ? data.extra_fields?.short_name
+      : data.metadata.accession;
+
+  const subtitleFullName =
     typeof data.metadata.name === 'string'
       ? data.metadata.name
       : data.metadata.name.name;
 
-  let desc: StructuredDescription | undefined = undefined;
-  let included: [string, Reference][] | undefined = undefined;
-  if (showDescription) {
-    const description = data.extra_fields?.description;
-    const literature = data.extra_fields?.literature;
-    if (description?.length) {
-      desc = description[0];
-      const citations = description2IDs(desc.text);
-      included = Object.entries(literature || {})
-        .filter(([id]) => citations.includes(id))
-        .sort(
-          (a, b) =>
-            (desc as StructuredDescription).text.indexOf(a[0]) -
-            (desc as StructuredDescription).text.indexOf(b[0]),
-        );
-    }
-  }
   return (
     <Card
       imageComponent={
```

Comment:
The `MemberSymbol` component isn't really informative, as it's simply a coloured database icon. Member database signatures also have a type, so I would prefer if 

https://github.com/ProteinsWebTeam/interpro7-client/blob/56637c3ea157cd10f82d374bc385629b97c24e2c/src/components/Entry/Card/index.tsx#L43-L59

was replaced by 

```ts
        <Tooltip title={`${data.metadata.type.replace('_', ' ')} type`}>
          <interpro-type
            dimension="2em"
            type={data.metadata.type.replace('_', ' ')}
            aria-label="Entry type"
          />
        </Tooltip>
```